test/integration/builder/additional.js

```diff
@@ -951,6 +908,7 @@ module.exports = function(knex) {
       const knexPrototype = Object.getPrototypeOf(knexDb.client);
       const originalWrappedCancelQueryCall =
         knexPrototype._wrappedCancelQueryCall;
+
       knexPrototype._wrappedCancelQueryCall = (conn) => {
```

Comment:
Hmm... we should probably be using a `sinon` sandbox here rather than patching the prototype ourselves.  This will help guarantee that the prototype is restored properly once the test case concludes.  (Although this is out of scope for this PR, it might be worth implementing this fix if it's easy to do)

cc: @kibertoad